exporter/sumologicexporter/config.go

```diff
@@ -69,6 +74,13 @@ func createDefaultClientConfig() confighttp.ClientConfig {
 
 func (cfg *Config) Validate() error {
 
+	switch cfg.CompressEncoding {
+	case configcompression.TypeGzip, configcompression.TypeDeflate, NoCompression:
+		fmt.Printf("support for compress encoding type %v has been deprecated, please use compression instead\n", cfg.CompressEncoding)
+	default:
```

Comment:
I would only check if `cfg.CompressEncoding` exist

Reply:
and if yes, I would raise an error that the option is deprecated

Reply:
ohh 😆 done!